CAM/CreateCamera.md

```diff
@@ -7,10 +7,47 @@ ns: CAM
 // 0x5E3CF89C6BCCA67D 0x5D6739AE
 Cam CREATE_CAMERA(Hash camHash, BOOL active);
 ```
+Creates a camera with the specified camera hash, You can use `SET_CAM_` natives to manipulate the camera.  
+Make sure to call [RENDER_SCRIPT_CAMS](#_0x07E5B515DB0636FC) once the camera is created, or this won't have any visible effect.
 
+```
+NativeDB Introduced: v323
+```
+
+### Camera hashes:
```

Comment:
Hey we have the camera names [here](https://docs.fivem.net/natives/?_0xC3981DCE61D9E13F)

Reply:
Hey, I know that, I just added the camera hashes beside the names

Reply:
I'll edit it as soon as I can 🐌